### ISSUE TITLE
roan help map . refers to options panel

### ISSUE THREAD

**Issue by kenfhill84083** [vboerchers/roan-presentation#1]
I assume when the roan help map refers to "options panel" it should be changed to "properties panel".

**Comment by vboerchers** [comment]
If you like you could improve the documentation maps and submit improved versions of the docs. However, my plan was to keep the add-on alive. The documentation is a bit out of my scope.

**Comment by kenfhill84083** [comment]
Fair enough.  I have two problems -- 1. I generally like to write about something I know about.  Like scripting, Roan is kind of confusing to me.  If someone with Roan experience wanted to get the ball rolling, I could contribute to the extent of my knowledge.  2. I don't have a lot of time right now to play around with Roan.  I'll keep it in mind and try to find some odd bits of time to learn and experiment.
